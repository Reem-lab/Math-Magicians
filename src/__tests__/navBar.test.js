import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from '../component/NavBar';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <NavBar />
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
