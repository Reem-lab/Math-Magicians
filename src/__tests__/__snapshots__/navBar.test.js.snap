// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="navBar"
>
  <h1>
    Math Magicians
  </h1>
  <ul
    className="ul-links"
  >
    <li
      className="li-links"
    >
      <a
        aria-current="page"
        className="links active-link"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        aria-current={null}
        className="links"
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        aria-current={null}
        className="links"
        href="/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </li>
  </ul>
</nav>
`;
